platform/darwin/src/MGLAccountManager.m

```diff
@@ -8,12 +8,14 @@
 @interface MGLAccountManager ()
 
 @property (atomic) NSString *accessToken;
+@property (nonatomic) NSURL *apiBaseURL;
```

Comment:
Do these need to be here, if it's already in `MGLAccountManager_Private.h`?

Reply:
🤔 When I don't include it, `apiBaseURL` can't be accessed from `MGLOfflinePack`.